### PR TITLE
media-gfx/cura: version bump to 4.3.0

### PR DESCRIPTION
**Pull request by Perlovka**
Could be built with gcc-9 or clang-8 now

### CODE REVIEW

File: dev-libs/libcharon/libcharon-4.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+LICENSE="LGPL-3"
+SLOT="0/3"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RDEPEND="${PYTHON_DEPS}"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
```
Comment (msva) [inline dev-libs/libcharon/libcharon-4.3.0.ebuild]: it also needs patching against python version hardcode

Comment (Perlovka) [reply dev-libs/libcharon/libcharon-4.3.0.ebuild]: It's already here, as additional patch.

**Comment by juippis** [comment]
No love for: 
  https://bugs.gentoo.org/693170
  https://bugs.gentoo.org/693176
  https://bugs.gentoo.org/693172
  https://bugs.gentoo.org/693166
?

**Comment by Perlovka** [comment]
@juippis I fixed some tests, but there are more to fix, have no time for now. Also some of the bugs I can not reproduce, need to set up clean environment.

**Comment by juippis** [comment]
Ok, thanks!

**Pull request closed without merging**